src/github.com/getlantern/balancer/balancer.go

```diff
@@ -17,14 +19,20 @@ const (
 )
 
 var (
+	// When Dial() is called after an idle period larger than
+	// RecheckAfterIdleFor, Balancer will recheck all dialers to make sure they
+	// are alive and have up-to-date metrics.
+	RecheckAfterIdleFor = 1 * time.Minute
```

Comment:
It implies that it can be changed.  To change it, we'd want to protect it with a mutex, right?